contracts/pcv/compound/CompoundPCVDepositBase.sol

```diff
@@ -38,15 +27,21 @@ abstract contract CompoundPCVDepositBase is PCVDeposit {
     function withdraw(address to, uint256 amountUnderlying)
         external
         override
-        onlyPCVController
+        hasAnyOfTwoRoles(TribeRoles.GOVERNOR, TribeRoles.PCV_CONTROLLER)
         whenNotPaused
     {
-        require(
-            cToken.redeemUnderlying(amountUnderlying) == 0,
-            "CompoundPCVDeposit: redeem error"
-        );
-        _transferUnderlying(to, amountUnderlying);
-        emit Withdrawal(msg.sender, to, amountUnderlying);
+        _withdraw(to, amountUnderlying);
+    }
+
```

Comment:
Add withdrawAllAvailable which pulls the lesser of the two: amount wanted to withdraw or available liquidity in compound